branchpro/apps/_simulation.py

```diff
@@ -75,15 +78,41 @@ def __init__(self):
                     dbc.Row([
                         dbc.Col(dcc.Graph(figure=self.plot.figure,
                                           id='myfig')),
-                        dbc.Col(self.sliders.get_sliders_div())],
-                        align='center'
+                        dbc.Col(
+                            self.sliders.get_sliders_div(), id='all-sliders')
+                            ],
+                            align='center'
+                            ),
+                    dcc.Upload(
+                        id='upload-data',
+                        children=html.Div([
+                            'Drag and Drop or ',
+                            html.A(
+                                'Select Files',
+                                style={'text-decoration': 'underline'})
```

Comment:
```suggestion
                                style={'text-decoration': 'underline'}),
                            ' to upload your Incidence Number data.'
```
What do you think of adding some sort of description like this to the field?

I also think we should have 1--2 sentences explaining how to upload data, where it will appear, and the required column names. This could go either in the top text or in an extra html div above or below the upload field.